make_preselection.py

```diff
@@ -252,12 +253,16 @@
             MhhvMhFailTrigdown.Sumw2()
 
 
-
+    print("Histograms booked")
     ###############################
     # Grab root file that we want #
     ###############################
-    file_string = Load_jetNano(options.set,options.year)
+    if len(options.set.split('-')) > 1:
+        file_string = Load_jetNano(options.set.split('-')[0]+'_'+options.set.split('-')[1],options.year)
```

Comment:
Yeah now that I look at this, I think the best thing to do is change how the hhTrees/ stuff behaves so that it has ttbar-semilep instead of ttbar_semilep. I'm going to git checkout the initial-Dev branch, make the change, and commit it to go in this PR